example-site/content-org/construct-hugo-front-matter-from-menu-meta-data.org

```diff
@@ -0,0 +1,12 @@
+#+HUGO_BASE_DIR: ../
+#+HUGO_FRONT_MATTER_FORMAT: yaml
+
+* Menu Meta Data in YAML Front Matter
+:PROPERTIES:
+:EXPORT_FILE_NAME: menu-meta-data-yaml
+:EXPORT_DATE: 2017/07/17
+:HUGO_MENU_NAME: main
+:HUGO_MENU_WEIGHT: 100
+:HUGO_MENU_PARENT: posts
```

Comment:
About this, I was thinking of changing the way of representing this in Org. Then a user can have a menu variable with any name they like (example: you had a custom var called identifier).

How about:

```
#+HUGO_MENU: :name main :weight 100 :parent posts :foo bar
```